styles.css

```diff
@@ -0,0 +1,95 @@
+* {
+  box-sizing: border-box;
+}
+
+/* nav bar */
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+}
+
+.ic-menu {
+  margin-right: 20px;
+  margin-top: -20px;
+}
+
+/* main */
+
+/* main {
+    display: flex;
+    flex-direction: column;
+  } */
+
+/* page break */
```

Comment:
- [ ] I request that you eliminate the comments in the code-base because it makes our code unreadable